plugins/dev-tools/src/toolboxCategories.js

```diff
@@ -1,340 +1,885 @@
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+            'TO': {
+              'shadow': {
+                'type': 'math_number',
+                'id': 'o(J=1}3OGC}o{WXU^~O?',
+                'fields': {
+                  'NUM': 10,
+                },
+              },
+            },
+            'BY': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'controls_forEach',
+          'kind': 'block',
+          'fields': {
+            'VAR': {
+              'name': 'j',
+            },
+          },
+        },
+        {
+          'type': 'controls_flow_statements',
+          'kind': 'block',
+          'enabled': false,
+          'fields': {
+            'FLOW': 'BREAK',
+          },
+        },
+      ],
+    },
+    {
+      'kind': 'category',
+      'name': 'Math',
+      'categorystyle': 'math_category',
+      'contents': [
+        {
+          'type': 'math_number',
+          'kind': 'block',
+          'fields': {
+            'NUM': 123,
+          },
+        },
+        {
+          'type': 'math_arithmetic',
+          'kind': 'block',
+          'fields': {
+            'OP': 'ADD',
+          },
+          'inputs': {
+            'A': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+            'B': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 1,
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'math_single',
+          'kind': 'block',
+          'fields': {
+            'OP': 'ROOT',
+          },
+          'inputs': {
+            'NUM': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 9,
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'math_trig',
+          'kind': 'block',
+          'fields': {
+            'OP': 'SIN',
+          },
+          'inputs': {
+            'NUM': {
+              'shadow': {
+                'type': 'math_number',
+                'fields': {
+                  'NUM': 45,
+                },
+              },
+            },
+          },
+        },
+        {
+          'type': 'math_constant',
+          'kind': 'block',
+          'fields': {
+            'CONSTANT': 'PI',
+          },
+        },
+        {
+          'type': 'math_number_property',
+          'kind': 'block',
+          'extraState': '<mutation divisor_input=\'false\'></mutation>',
```

Comment:
You should be able to remove any `extraState` properties that have XML in them. Here and below.